docs/data/material/customization/default-theme/DefaultTheme.js

```diff
@@ -64,13 +64,11 @@ const StyledSwitch = styled(Switch)(({ theme }) => [
     },
   },
 ]);
-
```

Comment:
We don't have any changed in this file, but we removed empty lines, let's fix this behavior.

Reply:
Restore the changes. This behavior is fixed by https://github.com/mui/material-ui/pull/42527